src/components/SearchResult/SearchResult.jsx

```diff
@@ -18,20 +18,18 @@ function SearchResult() {
             "day"
           );
           return (
-            <>
-              <TableBody
-                key={book.id}
-                id={book.id}
-                title={book.title}
-                firstName={book.firstName}
-                surName={book.surname}
-                email={book.email}
-                roomId={book.roomId}
-                checkInDate={book.checkInDate}
-                checkOutDate={book.checkOutDate}
-                stayNights={stayNightsTotal}
-              />
-            </>
+            <TableBody
+              key={book.id}
+              id={book.id}
+              title={book.title}
+              firstName={book.firstName}
+              surName={book.surname}
+              email={book.email}
+              roomId={book.roomId}
+              checkInDate={book.checkInDate}
+              checkOutDate={book.checkOutDate}
+              stayNights={stayNightsTotal}
+            />
```

Comment:
This is a matter of personal prefecence to an extent but, I would personally just give the whole book object to TableBody so it can do what it needs to do

Reply:
I will try to look this up after main tickets are done